packages/back-end/test/permissions.test.ts

```diff
@@ -2878,6 +2878,633 @@ describe("PermissionsUtilClass.canDeleteMetric check", () => {
   });
 });
 
+describe("PermissionsUtilClass.canViewCreateFactTableModal check", () => {
```

Comment:
These tests are a bit repetitive - but, I feel like they'll be really valuable as we explore custom roles. Though, we'll likely want to adjust/add more test cases when that happens.

Open to suggestions if anyone has a pattern they've seen before that can either automate some of these tests, or make this a bit more extensible.